testingfacade/APIs/schemas/QuestionSchema.json

```diff
@@ -0,0 +1,153 @@
+            "enum": ["single_choice", "multi_choice", "action"]
+        },
+        "question_id": {
+            "type": "string",
+            "description": "A unique string for each individual question sent. Where a test contains multiple questions they will share a name attribute but have a unique id."
+        },
+        "name": {
+            "type": "string",
+            "description": "The test name, indicating the test that sent the question."
+        },
+        "description": {
+            "type": "string",
+            "description": "A brief description of the test"
+        },
+        "question": {
+            "type": "string",
+            "description": "Human readable question text containing instructions to carry out the test"
+        },
+        "answers": {
+            "type": "array",
+            "description": "Array containing answer objects to be used to select the correct answer_id to be returned.",
+            "examples": [
+                [
+                    {
+                        "answer_id": "answer_2",
+                        "label": "s3/wright",
+                        "description": "Mock sender 3",
+                        "id": "71ce775c-cdc5-479a-bd29-87c081905ecf",
+                        "answer_str": "s3/wright (Mock sender 3, 71ce775c-cdc5-479a-bd29-87c081905ecf)"
+                    }
+                ]
+            ],
+            "items": {
+                "description": "Individual answer details, the label, description and id of the relevant resource and an associated answer_id",
+                "type": "object",
+                "required": [
+                    "answer_id",
+                    "label",
+                    "description",
+                    "id",
+                    "answer_str"
+                ],
+                "properties": {
+                    "answer_id": {
+                        "type": "string",
+                        "description": "ID unique to each answer within a question. This is the value to be returned for validation."
+                    },
+                    "label": {
+                        "type": "string",
+                        "description": "Resource label as might be displayed on the NCuT"
+                    },
+                    "description": {
+                        "type": "string",
+                        "description": "Resource description as might be displayed on the NCuT"
+                    },
+                    "id": {
+                        "type": "string",
+                        "description": "Resource ID as might be displayed on the NCuT"
+                    },
+                    "answer_str": {
+                        "type": "string",
+                        "description": "String combining all the resource details for display on the testing facade"
+                    }
+                }
+            }
+        },
+        "time_sent": {
+            "type": "string",
```

Comment:
Seems to be a number in previous examples? Cf. `time_answered`.

Reply:
Good spot, will update this once I've determined whether or not it can be removed completely or be changed to use TAI time to be consistent with the rest of the test suite.

Reply:
Changed this and timeout to type number. `time_sent` is staying in the Question API as it is needed to make sure the test suite and test facade stay in sync for question timeouts.

Reply:
Resolving; `time_sent` now also gone.